### PR TITLE
[READY] Add featured images to related posts below a blog post

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] add images and featured layout to related posts below a blog post 
- [x] #426 needs to be merged first 

I changed related to 5 posts, because 10 feels to much to me in this layout.

### CODE REVIEW

File: source/blog/_blog-article-preview.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+          <% if author && src = team_avatar_url(author) %>
+            <div class="author-avatar">
+              <%= image_tag src, alt: author_name, class: "avatar" %>
+            </div>
+          <% end %>
+          <div class="author">
+            <span class="author-name"><%= author_name %></span>
+            <time datetime="<%= article.date.strftime('%F') %>"><%= I18n.l(article.date, format: "%e %B %Y", locale: I18n.locale.to_s) %></time>
+          </div>
```
Comment (fatboypunk) [inline source/blog/_blog-article-preview.erb]: Maybe we could even make this whole showing an author thing a partial?

Comment (RonaldDijkstra) [inline source/blog/_blog-article-preview.erb]: 👍

Comment (RonaldDijkstra) [inline source/blog/_blog-article-preview.erb]: I should make it for showing author on the blog post itself too, but not in this PR!

**Comment by RonaldDijkstra** [comment]
## Now: 

blogpost-now

## Changed: 

blogpost-then

**Comment by RonaldDijkstra** [comment]
Aiming at keeping users longer and let them read more posts at our blog by making the more posts section more dynamic.

**Pull request merged**